src/neo-vm/InteropService.cs

```diff
@@ -6,6 +6,12 @@ namespace Neo.VM
     public class InteropService
     {
         private Dictionary<uint, Func<ExecutionEngine, bool>> dictionary = new Dictionary<uint, Func<ExecutionEngine, bool>>();
+        private Dictionary<string, uint> dictionaryStr = new Dictionary<string, uint>();
```

Comment:
Dictionary<string, Func<ExecutionEngine, bool>> dictionaryStr = new Dictionary<string, Func<ExecutionEngine, bool>>();

Reply:
In this case, I  will need string to.m uint, to be able to compare it inside Neo project.... example, if user passes uint 1000  I need to know if its a Storage.Put, to know the price.